### DISCUSSION TITLE
Fail create addressable when installing filament demo.

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5367] [Help]
Hi All,

I tried to install filament demo.
I got error at migration step.

```
2022_06_09_155042_create_addressable_table .............................................................................................. 0ms FAIL

  SQLSTATE[HY000]: General error: 1 near "as": syntax error (SQL: create table "addresses" ("id" integer not null primary key autoincrement, "country" varchar, "street" varcha
r, "city" varchar, "state" varchar, "zip" varchar, "full_address" varchar as (street  || ', ' || zip  || ' ' || city), "created_at" datetime, "updated_at" datetime))
```

Anyone know how to fix this ?
Thank You

**Comment by danharrin** [comment]
What DB are you using?

**Reply by altimusin** [reply]
Hi Danharrin,

I am using Sqlite
here is my phpinfo screenshoot

![image](https://user-images.githubusercontent.com/43429317/210092182-be66ca78-a163-4d7f-95b5-89efdfd52ce3.png)

**Reply by danharrin** [reply]
Try MySQL or Postgre

**Reply by altimusin** [reply]
the database successfully migrate dan seeded.

But when running the application, It direct me to \login address,. but showing 404.
I tried to access \admin,. but give me same 404 error.

![image](https://user-images.githubusercontent.com/43429317/210196777-05c912e6-e92b-4110-a0c9-a6c1f57c2ac2.png)

But, In with same server,. If I installing laravel+filament is running successfully.

any clue to fix this?

**Reply by danharrin** [reply]
no, it's definitely a server config issue

**Reply by altimusin** [reply]
Ok... You are right, there are issue in server config,. 
Now,. is fixed.

Thank you
